lib/fastly_nsq/listener.rb

```diff
@@ -116,8 +116,9 @@ def call(nsq_message)
     msg_info = {
       channel:  channel,
       topic:    topic,
-      attempts: nsq_message.attempts,
+      attempts: message.attempts,
```

Comment:
Message and nsq_message are the same then?

Reply:
we delegate attempts in `FastlyNsq::Message` to the nsq_mesage